test/framework/setup.go

```diff
@@ -119,6 +119,19 @@ name: %s
 	return YamlUniversal(mesh)
 }
 
+func Parallel(fns ...InstallFunc) InstallFunc {
+	return func(cluster Cluster) error {
+		eg := errgroup.Group{}
+		for _, fn := range fns {
+			installFn := fn
+			eg.Go(func() error {
+				return installFn(cluster)
```

Comment:
nit:
```suggestion
			eg.Go(func() error {
				return fn(cluster)
```

Reply:
`eg.Go` starts new goroutine. https://pauldigian.com/advanced-go-loops-and-goroutines

Technically fixed in 1.22 https://medium.com/@ksandeeptech07/improved-goroutine-handling-in-go-1-22-resolving-the-loop-variable-sharing-issue-691f6c9dd906 but I had the problem without it 🤔 I can double check